typescript/web/src/components/workspaces/workspaces-context.tsx

```diff
@@ -0,0 +1,45 @@
+import { Workspace } from "@labelflow/graphql-types";
+import { createContext, PropsWithChildren, useContext, useState } from "react";
+
+export type GqlWorkspace = Pick<
+  Workspace,
+  "id" | "name" | "slug" | "plan" | "image"
+>;
+export type WorkspacesProps = {
+  workspaces: GqlWorkspace[];
+  openCreateWorkspaceModal: () => void;
+};
+export type WorkspaceState = {
+  searchText: string;
+  setSearchText: (text: string) => void;
+  openCreateWorkspaceModal: () => void;
+  workspaces: GqlWorkspace[];
+  filteredWorkspaces: GqlWorkspace[];
+};
+const WorkspaceContext = createContext({} as WorkspaceState);
+
+export type WorkspacesContextProps = PropsWithChildren<WorkspacesProps>;
+export const WorkspacesContextProvider = ({
```

Comment:
```suggestion
export type WorkspacesContextProps = PropsWithChildren<WorkspacesProps>;

export const WorkspacesContextProvider = ({
```

I guess we have to search for a prettier setting which takes care of this for is (if that's possible?) 😅

Reply:
(Fixed)
Yes I would like to because I don't see those problems due to the cyclomatic complexity analyzer extension

Reply:
@Jordanlelay Actually, you can just disable the extra line (_which is called a "code-lens_) with `"codemetrics.basics.CodeLensEnabled": false`:

![image](https://user-images.githubusercontent.com/2091902/150993651-df3f0be4-383e-4885-acbc-5ebbd518fa71.png)

You'll lose the calculation details but it's not that useful and the settings can still be changed temporarily when you need it.

CC @valerian